unison-cli/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -3285,15 +3283,28 @@ displayNames unisonFile =
     (UF.typecheckedToNames unisonFile)
 
 diffHelper ::
-  Monad m =>
+  (Monad m) =>
   Branch0 m ->
   Branch0 m ->
   Action' m v (PPE.PrettyPrintEnv, OBranchDiff.BranchDiffOutput v Ann)
 diffHelper before after = do
+  currentRoot <- use LoopState.root
+  currentPath <- use LoopState.currentPath
+  diffHelperCmd currentRoot currentPath before after
+
+-- | A version of diffHelper that only requires a MonadCommand constraint
+diffHelperCmd ::
```

Comment:
There are still a bunch of places using `diffHelper`, but I needed a version which I could call within a `viewRemoteBranch`, so this is what I ended up with.